### ISSUE TITLE
Terminal buffer is not updating information

### ISSUE THREAD

**Issue by jeanbrazcosta** [microsoft/vscode#191825]
Type: <b>Bug</b>

In the insiders version, I noticed that the buffer is no longer automatically reflecting terminal information.

For example, when executing the mvn clean install command in my project, the information of the build that starts to run is updated from time to time in the buffer, sometimes it is only necessary to give a tab to go to the terminal and give a shift tab that I have the information updated while the build is running and printing various information.

In the insiders version, this update no longer occurs.

I have to keep pressing alt + f2, but I couldn't be sure if I still have all the updated information.

VS Code version: Code - Insiders 1.82.0-insider (35be9bf683eace09796e59d54f1f225bbc3a7866, 2023-08-30T05:33:47.972Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Remote OS version: Linux x64 5.15.90.1-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8365U CPU @ 1.60GHz (8 x 1896)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|31.72GB (11.44GB free)|
|Process Argv|--crash-reporter-id dd13835f-488a-4675-bdf9-21fedc4249ef|
|Screen Reader|yes|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: ubuntu|
|OS|Linux x64 5.15.90.1-microsoft-standard-WSL2|
|CPUs|Intel(R) Core(TM) i5-8365U CPU @ 1.60GHz (4 x 1896)|
|Memory (System)|15.62GB (5.22GB free)|
|VM|0%|
</details><details><summary>Extensions (22)</summary>

Extension|Author (truncated)|Version
---|---|---
remote-containers|ms-|0.307.0
remote-wsl|ms-|0.81.0
cucumber-official|Cuc|1.7.0
gitlens|eam|2023.8.3005
copilot|Git|1.106.367
copilot-chat|Git|0.7.2023083001
vscode-docker|ms-|1.26.0
vscode-language-pack-pt-BR|MS-|1.82.2023083009
postman-for-vscode|Pos|0.6.2
vscode-thunder-client|ran|2.10.5
java|red|1.21.0
intellicode-api-usage-examples|Vis|0.2.8
vscodeintellicode|Vis|1.2.30
vscode-boot-dev-pack|vmw|0.2.1
vscode-spring-boot|vmw|1.48.0
vscode-java-debug|vsc|0.54.0
vscode-java-dependency|vsc|0.23.1
vscode-java-pack|vsc|0.25.13
vscode-java-test|vsc|0.39.1
vscode-maven|vsc|0.42.0
vscode-spring-boot-dashboard|vsc|0.13.1
vscode-spring-initializr|vsc|0.11.2

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
vscod805:30301674
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30404738
py29gd2263:30784851
vsclangdf:30492506
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30618038
2i9eh265:30646982
showlangstatbar:30737417
0bi6i642:30823812
ecj1e332:30687743
pythonfmttext:30716741
fixshowwlkth:30771523
showindicator:30805243
pythongtdpath:30726887
i26e3531:30792625
gsofa:30797620
welcomedialog:30812478
pythonnosmt12:30779711
pythonidxpt:30768918
pythonnoceb:30776497
copilotsettingt:30808721
dsvsc013:30777762
dsvsc014:30777825
diffeditorv2:30786206
pythonlinttypecf:30823782
pythonmpsinfo:30815194
dsvsc015:30821418

```

</details>

**Comment by meganrogge** [comment]
@jeanbrazcosta `Shift+tab` no longer works and in fact breaks things in today's insider's, which will be fixed in tomorrow's build. When using `alt+f2` only, it is working for me. 

@jooyoungseo is this working for you?

**Comment by jeanbrazcosta** [comment]
@meganrogge shift+tab be deprecated in future releases?

If yes, I understand the change more sincerely I prefer shift + tab than pressing alt +f2 I think shift + tab is a faster way, but I understand if it is no longer possible.

Another thing I noticed is that when opening new terminals the buffer doesn't appear, I always have to press alt +f2 with each new terminal that opens.

**Comment by rperez030** [comment]
It appears to be working, but I have not run a longer session with something like building the sources for NVDA, for example. I have honestly never felt confident enough with the integrated terminal from VS Code, so I have tried to build my workflows by always launching an external terminal. I may give it a more serious try when entering commands become more stable.

**Comment by meganrogge** [comment]
@jeanbrazcosta yes, that is deprecated. You can still add it to your keybindings though.

> Another thing I noticed is that when opening new terminals the buffer doesn't appear, I always have to press alt +f2 with each new terminal that opens.

This is by design as one cannot input into the accessible buffer.

**Comment by meganrogge** [comment]
I can reproduce``

**Comment by meganrogge** [comment]
Interestingly, this works for me on mac in 1.82 but does not work in insider's.
